app/src/main/java/edu/uc/reedws/musiclink/IndividualPlaylistActivity.kt

```diff
@@ -3,7 +3,9 @@ package edu.uc.reedws.musiclink
 import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import kotlinx.android.synthetic.main.main_activity.*
+import kotlinx.android.synthetic.main.main_activity.libraryButton
+import kotlinx.android.synthetic.main.main_activity.searchButton
+import kotlinx.android.synthetic.main.main_activity.addPlaylistOrSongButton
```

Comment:
I removed the ' * ' from the imports because it is not a best practice to import all the imports under the library even if you don't need it. Let's say there is a java.awt.Event import which you didn't need but you still imported like this: java.awt.*. Now if you have your own Event import: reedws.edu.Event that you want to import, you won't be able to call Event object without writing the whole import location, which will lead to clutter in your codebase. 

You can also refer this community answer to understand more about import '*' issue here: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
@sukhbirsekhon I disagree with the * import thing. I agree that for 3rd party libraries * import is bad, but for our own activities it seems the Kotlin convention is to use * imports. If you run the code cleanup and reformat in android studio it will automatically convert activity imports to * imports

Reply:
@reediculous456 I think you have good point. I didn't know that when you clean up and reformat the code in Android studio, it converts imports to *. I never noticed it. I learned something new today.

Reply:
@reediculous456 I think you have a good point. I didn't know that when you clean up and reformat the code in Android studio, it converts imports to *. I never noticed it. I learned something new today.